masterlist.yaml

```
@@ -808,3 +808,28 @@ plugins:
       - 'NOM3.0.esp'
       - 'NOM3_PT SO.esp'
       - 'NoM_Creatures Loot Standard.esp'
+
+  ### Other Stuff (sort later) ###
+  - name: 'Better Propylon Teleport Warp.ESP'
+    url:
+      - link: 'https://www.nexusmods.com/morrowind/mods/46364/'
+        name: 'Pikas Miscellaneous Mods'
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Better Propylon Teleport Warp ESP' ]
+        condition: 'many("Better Propylon Teleport Warp(-Master Index)?\.esp")'
+  - name: 'Better Propylon Teleport Warp-Master Index.esp'
+    url:
+      - link: 'https://www.nexusmods.com/morrowind/mods/46364/'
+        name: 'Pikas Miscellaneous Mods'
+    after:
+      - 'master_index.esp'
+      - 'MasterIndexRedux.esp'
```

Plugins will automatically load after their requirements, so a load after for `master_index.esp` isn't necessary:

```suggestion
    after: [ 'MasterIndexRedux.esp' ]
```

---

Not all of the Morrowind mods actually have requirements. Or masterfiles at all.

---

Better Propylon Teleport Warp-Master Index.esp has only Morrowind.esm as masterfile and requirement

---

You added it as a requirement below, though, so it makes a load after redundant. See the `req` section of [this part](https://loot-api.readthedocs.io/en/latest/metadata/data_structures/plugin.html) of the metadata syntax docs.